similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientGenotype.java

```diff
@@ -48,9 +49,14 @@
  */
 public class DefaultPatientGenotype extends AbstractExome implements PatientGenotype
 {
+
     /** Factory for loading exome data. */
     protected static ExomeManager exomeManager;
 
+    /** List of genes variant interpretations of pathogenic status 3-5 (VUS or higher). */
+    private static final String[] TOP_INTERPRETATIONS =
+    {"Pathogenic", "Likely Pathogenic", "Variant of Unknown Significance"};
```

Comment:
These are translated strings, not the internal keys stored in the actual object. Which means that if PhenoTips is set in French, the value you get will be different. Why isn't the internal key used?